src/main/java/com/expense/Sulaah/service/GroupServiceImpl.java

```diff
@@ -22,57 +22,59 @@ public class GroupServiceImpl implements GroupService {
     @Autowired
     private UserRepository userRepository;
 
+	private Group group;
+
     @Override
-    public Group createGroup(int userId, Group group) {
+    public Group createGroup(int userId, String name) throws SecurityException {
         Optional<User> user = userRepository.findById(userId);
-        if (user.isPresent()) {
-            List<User> usersInGroupList = group.getUsersInGroup();
-            if (usersInGroupList == null) {
-                usersInGroupList = new ArrayList<User>();
-                group.setUsersInGroup(usersInGroupList);
-            }
-            usersInGroupList.add(user.get());
-            return groupRepository.save(group);
-        } else {
-            return null;
+        if (!user.isPresent()) {
+			throw new SecurityException("User not present with given id: " + userId);
```

Comment:
why security exception!

Reply:
If user is not present, then how it creates group so it is security concern